nginx/config/nginx.conf

```diff
@@ -29,15 +29,15 @@ http {
         server_name  _;
         return 301 https://$host$request_uri;
     }
-    {{else}}
+    {{~ else ~}}
```

Comment:
The convention in core-plans thusfar has been a single tilde at the end https://github.com/habitat-sh/core-plans/blob/master/elasticsearch/config/elasticsearch.yml#L52